geometry.go

```diff
@@ -40,6 +40,17 @@ func NewSize(w int, h int) Size {
 	return Size{w, h}
 }
 
+// PixelSize describe an object width and height in Pixels
```

Comment:
I don't think that we should add a new public API for this as it's an internal detail.
If you think that our driver needs this type then please move it out of the public API.

Reply:
I moved it to internal/geometry.go.